packages/react-component-library/src/components/DatePicker/partials/StyledDayPicker.tsx

```diff
@@ -196,4 +198,20 @@ export const StyledDayPicker = styled(DayPicker)<StyledDayPickerProps>`
     color: ${color('neutral', '200')};
     pointer-events: none;
   }
+
+  .DayPicker-TodayButton {
+    ${getButtonStyles({
+      $size: COMPONENT_SIZE.SMALL,
+      $variant: BUTTON_VARIANT.TERTIARY,
+    })};
+    width: 100%;
+    margin-bottom: ${spacing('6')};
+    box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.5);
```

Comment:
We shouldn't have the box-shadow anymore - the NSS screenshot is old. So feel free to remove this.

Reply:
Lets make the buttons look just like the Tertiary variant.

![Screenshot 2024-06-24 at 10 21 51](https://github.com/Royal-Navy/design-system/assets/48086589/49b4a0cb-bfa4-4efe-99d7-bbbba1a89f4e)